tests/examples_arguments_syntax/line_chart_with_cumsum_faceted.py

```diff
@@ -0,0 +1,36 @@
+"""
+Faceted Line Chart with Cumulative Sum
+------------------------------
+This chart creates multiple line subcharts from the cumulative sum of a field, one for each category.
```

Comment:
```suggestion
This chart creates one facet per natural disaster and shows the the cumulative number of deaths for that category.
Note the use of different predicates to filter based on both a list and a range.
```

Same in the other file

Reply:
- Added details to example description, calling out the use of different predicates as filters